cs/src/Contracts/ClusterDetails.cs

```diff
@@ -0,0 +1,15 @@
+namespace Microsoft.VsSaaS.TunnelService.Contracts;
+
+/// <summary>
+/// Tunnel service cluster details.
+/// </summary>
+public class ClusterDetails {
+    /// <summary>
+    /// A cluster identifier based on its region.
+    /// </summary>
+    public string? ClusterId { get; set; }
+    /// <summary>
+    /// The cluster DNS host.
+    /// </summary>
+    public string? Host { get; set; }
```

Comment:
How about `Hostname` so it is less easily confused with a tunnel "host"?

Reply:
Or should it be a full `https://` URI?

Reply:
I had it with https before which was nicer in the CLI, but I thought host name might be better for URI builders. But URI builders are probably happy taking either form anyway.

Reply:
Updated to use full uri.